pkg/tsdb/elasticsearch/client/index_pattern.go

```diff
@@ -86,6 +92,22 @@ func (ip *dynamicIndexPattern) GetIndices(timeRange *tsdb.TimeRange) ([]string,
 	return indices, nil
 }
 
+/// PPL currently does not support multi-indexing through lists, so a wildcard
+// pattern is used to match all patterns and relies on the time range filter
+// to filter out the incorrect indecies.
+func (ip *dynamicIndexPattern) GetPPLIndex() (string, error) {
```

Comment:
Have you look at https://www.elastic.co/guide/en/elasticsearch/reference/current/date-math-index-names.html.
PPL can support that. 
```
source = `<log-{now/d}>`
```

Reply:
didn't know about this. Will look into that!